packages/docs/src/examples/v-progress-linear/misc-normalized.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-container>
+    <v-progress-linear
+      v-for="(percentage, index) of percentages"
+      :key="index"
+      :model-value="percentage"
+      color="blue-grey"
+      height="25"
+    >
+      <template v-slot:default="{ value }">
+        <strong>
+          {{ value }}%
```

Comment:
The easiest thing that comes to mind when looking at this example is `.toFixed()` or  `.toFixed(1)`.